packages/app/cypress/e2e/cypress-in-cypress-component.cy.ts

```diff
@@ -29,7 +29,7 @@ describe('Cypress In Cypress CT', { viewportWidth: 1500, defaultCommandTimeout:
 
       snapshotAUTPanel('browsers open')
       cy.contains('Canary').should('be.hidden')
-      cy.contains('The viewport determines the width and height of your application. By default the viewport will be 500px by 500px for Component Testing unless specified by a cy.viewport command.')
+      cy.contains(cy.i18n.runner.viewportTooltip.infoText.replace('{0}', '500px').replace('{1}', '500px').replace('{2}', 'component'))
```

Comment:
There was never consensus among the engineering team to move forward with using cy.i18n replacement values for text in the tests. There were quite a few people opposed to this convention since readability is hindered.

Reply:
Oooh does this mean I should use a hardcoded string?

Reply:
There never was a consensus - IIRC we ended up with "as feels appropriate to you."

Reply:
It's a tradeoff, should the test "know" the correct text, or should the content json be the source of truth? Tyler called it "asserting if the pipes work" vs "asserting what goes through them" or something like that, which I liked.

It's more readable with plain strings for sure - especially this example, I would agree with keeping it as a plain string here.

I've kinda come around to thinking component tests should always refer to the content from the json, to confirm the component is correctly hooked up to its source of content, and E2E tests might benefit from using plain strings to confirm the user-facing text is what is expected. There are other considerations but this would at least get all the advantages of both approaches, and a clear boundary between them. We haven't revisited the discussion in a while though.